libs/ledger-live-common/.unimportedrc.json

```diff
@@ -356,6 +364,12 @@
     "deprecated-react-native-prop-types",
     "event-target-shim",
     "hoist-non-react-statics",
+    [
+      "just-curry-it",
```

Comment:
Can you explain in PR description why we are adding those 3 elements to unresolved dependencies ?

Is it because a dependency is unused or we are importing only types ?

Reply:
To be honest I don't know the CI failed for some reason, I ran `npx unimported -u` and that's the output